Makefile

```diff
@@ -166,6 +170,12 @@ $(BINDIR)/informer-gen:
 		--input-dirs "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1" \
 		--listers-package "github.com/kubernetes-incubator/service-catalog/pkg/client/listers" \
 		--output-package "github.com/kubernetes-incubator/service-catalog/pkg/client/informers"
+	# generate openapi
+	$(DOCKER_CMD) $(BINDIR)/openapi-gen \
+		--v 1 --logtostderr \
+		--go-header-file "vendor/github.com/kubernetes/repo-infra/verify/boilerplate/boilerplate.go.txt" \
+		--input-dirs "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1,k8s.io/kubernetes/pkg/api/v1,k8s.io/kubernetes/pkg/apis/meta/v1" \
+		--output-package "github.com/kubernetes-incubator/service-catalog/pkg/openapi"
```

Comment:
Is this the standard "openapi" package that upstream k8s uses? We want to match.